apps/admin/backend/src/util/cast_vote_records.test.ts

```diff
@@ -143,3 +148,48 @@ test('multiple flags', () => {
     hasWriteIn: true,
   });
 });
+
+test('updateMarkScoreDistributionFromMarkScores adds scores below 0.2 to correct buckets and ignores 0.0', () => {
+  const dist: MarkScoreDistribution = {
+    distribution: new Map<number, number>(),
+    total: 0,
+  };
+
+  const markScores: Tabulation.MarkScores = {
+    'contest-a': {
+      'option-1': 0.01,
+      'option-2': 0.15,
+      'option-3': 0.0, // should be ignored
+      'option-4': 0.21, // should not go in a bucket, but counts toward total
```

Comment:
I'm not sure whether actual mark scores have fixed-digit values like this but might be good to throw in some "longer" values, e.g., 0.12312341234, to test the rounding logic

Reply:
Oh perfect! So long as the test case is representative of reality, I have no issue here

Reply:
>  So long as the test case is representative of reality

Thanks for naming that principle!